### PR TITLE
Clarify documentation.

### PR DESCRIPTION
**Pull request by benpatterson**
Need better documentation for the custom firefox path capability.

### CODE REVIEW

**Comment by benpatterson** [comment]
@jzoldak @clytwynec @raeeschachar

**Comment by clytwynec** [comment]
Looks good to me. :+1:

File: bok_choy/browser.py [hunk 1]
```diff
@@ -127,10 +127,15 @@ def browser(tags=None, proxy=None):
     There are three cases:
 
     1. Local browsers: If the proper environment variables are not all set for the second case,
-        then we use a local browser.  The environment variable `SELENIUM_BROWSER` can be set to
-        specify which local browser to use, but the default is firefox.  Additionally, if a proxy
+        then we use a local browser.
+
+        * The environment variable `SELENIUM_BROWSER` can be set to
+        specify which local browser to use, but the default is firefox.
```
Comment (jzoldak) [inline bok_choy/browser.py]: As long as you are moving around this sentence, I'd change the "but" to an "and" or make that 2 sentences like "...which local browser to use. The default is Firefox."

**Comment by jzoldak** [comment]
:+1:

**Comment by benpatterson** [comment]
Taking Jay's comment, squashing, and then will merge.

**Pull request merged**